devicetypes/smartthings/zigbee-battery-accessory-dimmer.src/zigbee-battery-accessory-dimmer.groovy

```diff
@@ -332,15 +333,21 @@ def installed() {
 }
 
 def configure() {
-	sendEvent(name: "checkInterval", value: 2 * 60 * 60 + 10 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
+	// IKEA dimmer appears to only send us battery events when it is being used, so don't enroll in device health
+	if (isIkeaDimmer()) {
+		sendEvent(name: "DeviceWatch-Enroll", value: JsonOutput.toJson([protocol: "zigbee", scheme: "untracked"]), displayed: false)
+	} else {
+		sendEvent(name: "checkInterval", value: 2 * 60 * 60 + 10 * 60, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"], displayed: false)
+	}
 
 	if (isCentraliteSwitch()) {
 		zigbee.addBinding(zigbee.ONOFF_CLUSTER) + zigbee.addBinding(zigbee.LEVEL_CONTROL_CLUSTER) +
 			zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, BATTERY_VOLTAGE_ATTR) +
-			zigbee.configureReporting(zigbee.POWER_CONFIGURATION_CLUSTER, BATTERY_VOLTAGE_ATTR, DataType.UINT8, 0, 21600, null)
+			zigbee.batteryConfig(0, 21600, null)
 	} else {
 		zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, BATTERY_PERCENT_ATTR) +
-			zigbee.configureReporting(zigbee.POWER_CONFIGURATION_CLUSTER, BATTERY_PERCENT_ATTR, DataType.UINT8, 30, 10 * 60, null)
+			// Report no more frequently than 30 seconds, no less frequently than 6 hours, and when there is a change of 10% (expressed as half percents)
+			zigbee.configureReporting(zigbee.POWER_CONFIGURATION_CLUSTER, BATTERY_PERCENT_ATTR, DataType.UINT8, 30, 21600, 20)
```

Comment:
The report interval is longer than the checkInterval which will make the sengled switch get marked offline.

Reply:
I'm going to up the checkInterval, if that sounds good. I think 6 hours is plenty short enough, especially for battery devices.

Reply:
Does the sengled respond to pings? If both the report interval and check interval are 6 hours then one missed report could cause it to go offline if it doesn't respond to pings.

Reply:
I was going to set the checkInterval to 2 * reportInterval + ~2 mins to 10 mins. Unless you think reportInterval should be 3 hours and checkIntervale 6 hours + buffer? Or shorter?

Reply:
I do think `checkInterval` should be 2 * reportInterval + 2-10 minutes. Typically we'd do 5 minutes for report interval but that impacts battery life so I think it's a good idea to increase it. I'd say go with a 3 hour reportInterval and 6:10 checkInterval for now and also run it by product for their thoughts.

Reply:
Made some updates. I realized the IKEA dimmer does report to us, we just can't ping it. So now the reportInterval and checkInterval are all in sync. I'm surprised that some of it wasn't caught before. Anyway, I have this running at home, so we'll see if it marks offline.